internal/helm/registry/client.go

```diff
@@ -48,3 +56,11 @@ func ClientGenerator(isLogin bool) (*registry.Client, string, error) {
 	}
 	return rClient, "", nil
 }
+
+func clear(file string) error {
+	err := os.Remove(file)
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
Looks like this function isn't doing much different from where it's used, same kind of redundant error check. Maybe move it in-line?
And the multi-error can use the k8s apimachinery aggregate error above.